app/controllers/users/pulls/review_requests_controller.rb

```diff
@@ -2,6 +2,10 @@ class Users::Pulls::ReviewRequestsController < ApplicationController
   def new
     @pull = current_user.pulls.find(params[:pull_id])
     @review_req = current_user.review_requests.build(name: @pull.name, description: @pull.description, pull_id: @pull.id)
+
+    # pullとroomの選択肢
+    @pull_collection = current_user.pulls.where(is_open: true).order(created_at: :desc).map { |pull| [pull.name, pull.id] }
+    @room_collection = current_user.participating_rooms.map { |room| [room.name, room.id] }
```

Comment:
__[RuboCop]__

Layout/SpaceInsideBlockBraces: Space between { and | detected.


<!-- Comment by [SideCI](https://sideci.com) -->